app/angular/src/server/angular-cli_config.js

```diff
@@ -0,0 +1,69 @@
+const path = require('path');
+const fs = require('fs');
+
+export function getAngularCliWebpackConfigOptions(dirToSearch, appIndex = 0) {
+  const fname = path.join(dirToSearch, '.angular-cli.json');
+  if (!fs.existsSync(fname)) {
+    return null;
+  }
+  const cliConfig = JSON.parse(fs.readFileSync(fname, 'utf8'));
+  if (!cliConfig.apps || !cliConfig.apps.length) {
+    throw new Error('.angular-cli.json must have apps entry.');
+  }
+  const appConfig = cliConfig.apps[appIndex];
+
+  // FIXME dummy value
```

Comment:
Which of these are dummy values?

Reply:
it's `outputPath: 'outPutpath'`. I'll amend comment 🙇